### PR TITLE
describe model load and convertion to IR

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
more detailed description of how to download and convert model to IR

### CODE REVIEW

File: models/public/index.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ OpenVINO&trade; toolkit provides a set of public models
 that you can use for learning and demo purposes or for developing deep learning
 software. Most recent version is available in the [repo on Github](https://github.com/openvinotoolkit/open_model_zoo).
 
-The models can be downloaded via Model Downloader
-(`<OPENVINO_INSTALL_DIR>/deployment_tools/open_model_zoo/tools/downloader`).
+The models can be downloaded and converted to OpenVINO Inference Engine format (\*.xml + \*.bin) using Model Downloader and other automation tools.
+Review tools description in the file `<OPENVINO_INSTALL_DIR>/deployment_tools/open_model_zoo/tools/downloader/README.md` for the details.
```
Comment (aalborov) [inline models/public/index.md:8]:
```suggestion
Review the tools description in the file `<OPENVINO_INSTALL_DIR>/deployment_tools/open_model_zoo/tools/downloader/README.md` for details.
```

**Review by aalborov — CHANGES_REQUESTED** [review]

**Review by aalborov — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/public/yolo-v4-tf/yolo-v4-tf.md [hunk 2]
```diff
@@ -94,6 +94,20 @@ Detection box has format [`x`,`y`,`h`,`w`,`box_score`,`class_no_1`, ..., `class_
...
+You can download models and if necessary convert them into Inference Engine format using the OpenVINO™ [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.
+
+An example of using the Model Downloader:
+```
+python3 downloader.py --name <model_name>
+```
+
+An example of using the Model Converter:
+```
+python3 converter.py --name <model_name>
```
Comment (eaidova) [inline models/public/yolo-v4-tf/yolo-v4-tf.md:108]: should not it be info for specific model for each model description?
(e.g. in yolo-v4-tf provide commands for yolo-v4-tf and so on)

**Review by andrew-zaytsev — APPROVED** [review]

File: models/public/yolo-v4-tf/yolo-v4-tf.md (hunk 2 above)
Comment (vladimir-dudnik) [reply models/public/yolo-v4-tf/yolo-v4-tf.md:108]: seems better to keep this generic, would easier to update phrase if /when needed

File: demos/interactive_face_detection_demo/cpp/README.md [hunk 3]
```diff
@@ -94,9 +94,19 @@ Options:
 
 Running the application with an empty list of options yields the usage message given above and an error message.
 
-To run the demo, you can use public or pre-trained models. To download the pre-trained models, use the OpenVINO [Model Downloader](../../../tools/downloader/README.md). The list of models supported by the demo is in [models.lst](./models.lst).
+To run the demo, you can use public or Intel pre-trained models. You can download models and if necessary convert them into Inference Engine format using the OpenVINO™ [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below. The models which are supported by the demo are listed in the [models.lst](./models.lst) file.
```
Comment (IRDonch) [inline demos/interactive_face_detection_demo/cpp/README.md:97]:
```suggestion
To run the demo, you can use public or Intel pre-trained models. You can download models and, if necessary, convert them into Inference Engine format using the OpenVINO™ [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below. The models which are supported by the demo are listed in the [models.lst](./models.lst) file.
```

File: demos/interactive_face_detection_demo/cpp/README.md [hunk 4]
```diff
@@ -94,9 +94,19 @@ Options:
...
 
-> **NOTE**: Before running the demo with a trained model, make sure the model is converted to the Inference Engine format (\*.xml + \*.bin) using the [Model Optimizer tool](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html).
+An example of using the Model Downloader:
+```
+python3 downloader.py --list models.lst
```
Comment (IRDonch) [inline demos/interactive_face_detection_demo/cpp/README.md:101]: This is an impossible command, as it implies that `downloader.py` and `models.lst` are in the same directory, which is impossible. You should probably use the full path for the downloader.

File: demos/interactive_face_detection_demo/cpp/README.md [hunk 5]
```diff
@@ -94,9 +94,19 @@ Options:
...
+```
+
+An example of using the Model Converter:
+```
+python3 converter.py --list models.lst
+```
+
+> **NOTE**: Before running the demo with a trained model, make sure the model is downloaded and converted to the Inference Engine format (\*.xml + \*.bin) using OpenVINO™ [Model Downloader and other automation tools](../../../tools/downloader/README.md).
```
Comment (IRDonch) [inline demos/interactive_face_detection_demo/cpp/README.md:109]: Maybe just delete this note? It seems that it just duplicates the information above.

File: models/intel/icnet-camvid-ava-0001/description/icnet-camvid-ava-0001.md [hunk 6]
```diff
@@ -6,7 +6,19 @@ A trained model of ICNet for fast semantic segmentation, trained on the CamVid\*
...
+- Pole
+- Road
+- Pavement
+- Tree
+- SignSymbol
+- Fence
+- Vehicle
+- Pedestrian
+- Bike
+- Unlabeled
```
Comment (IRDonch) [inline models/intel/icnet-camvid-ava-0001/description/icnet-camvid-ava-0001.md:21]: Maybe you could add this list as a file under `data/dataset_classes` instead?

File: models/public/aclnet/aclnet.md [hunk 7]
```diff
@@ -49,18 +49,32 @@ Audio, name - `input`, shape - `1,1,1,L`, format is `N,C,1,L` where:
...
+
+An example of using the Model Downloader:
+```
+python3 downloader.py --name <model_name>
+```
+
+An example of using the Model Converter:
+```
+python3 converter.py --name <model_name>
+```
```
Comment (IRDonch) [inline models/public/aclnet/aclnet.md:76]: Repeating this in every model page seems questionable. Maybe it could be added to `models/public/index.md` instead?

File: models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md [hunk 8]
```diff
@@ -208,6 +208,20 @@ is an integer number encoding a character according to this dictionary:
         68 Y
         69 Z
 
+## Download a Model and Convert it into OpenVINOâ„¢ Inference Engine Format
+
+You can download models and if necessary convert them into Inference Engine format using the OpenVINOâ„¢ [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.
```
Comment (IRDonch) [inline models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md:213]: Corrupted characters here.

**Review by IRDonch — CHANGES_REQUESTED** [review]

File: models/intel/icnet-camvid-ava-0001/description/icnet-camvid-ava-0001.md (hunk 6 above)
Comment (vladimir-dudnik) [reply models/intel/icnet-camvid-ava-0001/description/icnet-camvid-ava-0001.md:21]: for segmentation models demo not using class labels, we only use mapping of classes to color and there is appropriate file with such mapping for CamVid but it is a bit different story

File: models/public/aclnet/aclnet.md (hunk 7 above)
Comment (vladimir-dudnik) [reply models/public/aclnet/aclnet.md:76]: we actually already have this info (how to load and convert model) in automation tools description. Might be better (from one prospective) to remove examples at all. @andrew-zaytsev what do you think?

File: models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md (hunk 8 above)
Comment (vladimir-dudnik) [reply models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md:213]: can we use `&trade;` symbols everywhere instead of those, directly coded in some codepage?

File: demos/interactive_face_detection_demo/cpp/README.md (hunk 5 above)
Comment (andrew-zaytsev) [reply demos/interactive_face_detection_demo/cpp/README.md:109]: Agree. Note should be removed

File: models/public/aclnet/aclnet.md (hunk 7 above)
Comment (andrew-zaytsev) [reply models/public/aclnet/aclnet.md:76]: I'd keep the examples in the model doc files. Looks like they are not going to be changed frequently, but they highlight it again that the user should convert the model before usage. Also, the user can see how to quickly convert the model in place without need to jump to another document.

File: models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md (hunk 8 above)
Comment (andrew-zaytsev) [reply models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md:213]: Let's remove `OpenVINO` from headings and Model Downloader name:

- Download a Model and Convert it into OpenVINOâ„¢ Inference Engine Format --> Download a Model and Convert it into Inference Engine Format 
- OpenVINO(tm) Model Downloader --> Model Downloader

File: models/intel/icnet-camvid-ava-0001/description/icnet-camvid-ava-0001.md (hunk 6 above)
Comment (IRDonch) [reply models/intel/icnet-camvid-ava-0001/description/icnet-camvid-ava-0001.md:21]: OK.

File: models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md (hunk 8 above)
Comment (IRDonch) [reply models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md:213]: I agree with @andrew-zaytsev, but I'll answer the question anyway:

> can we use `&trade;` symbols everywhere instead of those, directly coded in some codepage?

I slightly prefer using the real characters, because it looks better in the text editor. Our CI ensures that all files are encoded with UTF-8, so you can't mess up the encoding - the only reason these characters are wrong is because you inserted the wrong characters to begin with.

I don't see any reason to mass-replace existing non-ASCII characters with character entity references (like `&trade;`), although I don't mind if individual authors use them.

Comment (andrew-zaytsev) [reply models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md:213]: Agree with Roman, I also prefer using the real characters (in the openvino repo we do)

File: demos/interactive_face_detection_demo/cpp/README.md (hunk 4 above)
Comment (vladimir-dudnik) [reply demos/interactive_face_detection_demo/cpp/README.md]: changed

File: demos/interactive_face_detection_demo/cpp/README.md (hunk 5 above)
Comment (vladimir-dudnik) [reply demos/interactive_face_detection_demo/cpp/README.md]: removed

File: models/public/aclnet/aclnet.md (hunk 7 above)
Comment (vladimir-dudnik) [reply models/public/aclnet/aclnet.md]: add <omz_dir>/tools/downloader a a path

File: models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md (hunk 8 above)
Comment (vladimir-dudnik) [reply models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md]: applied suggested changes

**Comment by IRDonch** [comment]
Some of the links to the downloader README are incorrect; check the documentation build log.

File: models/public/forward-tacotron/description/forward-tacotron.md [hunk 9]
```diff
@@ -96,6 +96,20 @@ where:
    - T - time in mel-spectrogram
    - C - number of mels in mel-spectrogram
 
+## Download a Model and Convert it into Inference Engine Format
+
+You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.
```
Comment (anzhella-pankratova) [inline models/public/forward-tacotron/description/forward-tacotron.md:101]:
```suggestion
You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../../tools/downloader/README.md) as shown in the examples below.
```

File: models/public/human-pose-estimation-3d-0001/description/human-pose-estimation-3d-0001.md [hunk 10]
```diff
@@ -33,6 +33,20 @@ Expected color order is BGR.
 
 The net outputs three blobs with the following shapes: `[1, 57, 32, 56]`, `[1, 19, 32, 56]`, and `[1, 38, 32, 56]`. The first blob contains coordinates in 3D space, the second blob contains keypoint heatmaps and the third blob is keypoint pairwise relations (part affinity fields).
 
+## Download a Model and Convert it into Inference Engine Format
+
+You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.
```
Comment (anzhella-pankratova) [inline models/public/human-pose-estimation-3d-0001/description/human-pose-estimation-3d-0001.md:38]:
```suggestion
You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../../tools/downloader/README.md) as shown in the examples below.
```

File: models/public/open-closed-eye-0001/description/open-closed-eye-0001.md [hunk 11]
```diff
@@ -32,5 +32,19 @@ Fully convolutional network for recognition of eye state ('open', 'closed').
 1. name: "19", shape: [1, 2, 1, 1] - Softmax output across 2 type classes [open, closed]
 
 
+## Download a Model and Convert it into Inference Engine Format
+
+You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.
```
Comment (anzhella-pankratova) [inline models/public/open-closed-eye-0001/description/open-closed-eye-0001.md:37]:
```suggestion
You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../../tools/downloader/README.md) as shown in the examples below.
```

File: models/public/single-human-pose-estimation-0001/description/single-human-pose-estimation-0001.md [hunk 12]
```diff
@@ -50,6 +50,20 @@ The net outputs list of tensor. Count of list elements is 6. Every tensor with s
 
 The net outputs tensor with shapes: [1x17x48x36]. ( For every keypoint own heatmap)
 
+## Download a Model and Convert it into Inference Engine Format
+
+You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.
```
Comment (anzhella-pankratova) [inline models/public/single-human-pose-estimation-0001/description/single-human-pose-estimation-0001.md:55]:
```suggestion
You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../../tools/downloader/README.md) as shown in the examples below.
```

File: models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md [hunk 13]
```diff
@@ -76,6 +76,20 @@ The net outputs a blob with the shape: [1, 1, N, 7], where N is the number of de
     - (`x_min`, `y_min`) - coordinates of the top left bounding box corner
     - (`x_max`, `y_max`) - coordinates of the bottom right bounding box corner.
 
+## Download a Model and Convert it into Inference Engine Format
+
+You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.
```
Comment (anzhella-pankratova) [inline models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md:81]:
```suggestion
You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../../tools/downloader/README.md) as shown in the examples below.
```

File: models/public/wavernn/description/wavernn.md [hunk 14]
```diff
@@ -101,7 +101,21 @@ Note: B - batch size.
 
 Note: B - batch size.
 
-### Legal Information
+## Download a Model and Convert it into Inference Engine Format
+
+You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.
```
Comment (anzhella-pankratova) [inline models/public/wavernn/description/wavernn.md:106]:
```suggestion
You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../../tools/downloader/README.md) as shown in the examples below.
```

**Review by anzhella-pankratova — CHANGES_REQUESTED** [review]

**Review by anzhella-pankratova — APPROVED** [review]

**Review by IRDonch — APPROVED** [review]
